packages/typescript-configs/README.md

```diff
@@ -65,41 +53,34 @@ Here is an example of a tsconfig.json file extending the Workleap TypeScript con
 }
 ```
 
-**Tsconfig.json used for building**
 
-Building your TypeScript code involves compiling it down to JavaScript that can be run in a browser or server environment. This process may require different settings than checking compilation errors, such as a different target version or module format. By creating a separate tsconfig.json file for building, you can ensure that your code is compiled optimally for the target environment.
+### Monorepo Root
 
-Here is an example of a tsconfig.json file extending the Workleap TypeScript configuration for building:
+Before you can build your TypeScript code, you need to make sure that there are no compilation errors. This process involves running the TypeScript compiler in a "linting" mode that only checks for errors, without actually compiling the code.
+
+Here is an example of a tsconfig.json file extending the Workleap TypeScript configuration for linting:
 
 ```json
 {
-  "$schema": "https://json.schemastore.org/tsconfig",
-  "extends": "@workleap/library/build.json",
+  "extends": "@workleap/typescript-configs/monorepo-root",
   "compilerOptions": {
     "baseUrl": ".",
-    "outDir": "dist",
   },
   "include": ["src"],
   "exclude": ["node_modules"]
 }
 ```
 
-**Tsconfig.json used for building in dev mode**
+## Migrating from existing projects
 
-When you're developing your TypeScript code, you may want to compile it in a way that prioritizes speed over optimization. For example, you might want to use a watch mode that compiles your code quickly and automatically whenever you make changes. By creating a separate tsconfig.json file for building in dev mode, you can optimize the compilation process for speed and convenience, without sacrificing the quality of the resulting code.
-
-Here is an example of a tsconfig.json file extending the Workleap TypeScript configuration for building in dev mode:
+The configurations exported via this package are designed to be used by ESM projects. If you are migrating an existing project and you don’t want to migrate your project to ESM, you will need to do the following changes to your TypeScript configuration:
```

Comment:
I like what you did here.

I think it's also important to emphasis in the text that we expect that there would be other config values to override while migrating the project and that it's alright. This is the migration strategy that we are putting forward.

As they progress in their migration, they should be able to remove one by one the overrided config values.

Also, not sure I would add the `module: "commonjs"`.., It might not be the solution for every product as we move forward.

Reply:
i've added a description at the top that puts an emphasis on overriding the configuration during a migration. 

Also, i would keep the commonjs bit as it's the bit of information that is most important to change if one of our application were to use this config.